### PR TITLE
S02 us2 home page section2

### PR DESCRIPTION
**Pull request by pjroggy**

### CODE REVIEW

**Review by PaulBOURY — APPROVED** [review]

File: client/src/components/HomeSlider/Homeslider.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+.swiper-pagination-bullet {
+  background-color: $white-color;
+  opacity: 1;
+  transform: translateY(-0.3rem);
+}
+
+.swiper-pagination-bullet-active {
+  background-color: transparent;
+  background-image: url("../../assets/images/pacman-svgrepo-com.svg");
```
Comment (FGAT3) [inline client/src/components/HomeSlider/Homeslider.scss]: this image should be in "server/public/assets/images/icon" and change this path to "@harmonia/server/public/assets/images/icon/pacman-svgrepo-com.svg"

File: client/src/components/HomeSlider/Homeslider.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  background-size: cover;
+  background-position: center;
+  width: 1.2rem;
+  height: 1.2rem;
+  transform: translateY(0);
```
Comment (FGAT3) [inline client/src/components/HomeSlider/Homeslider.scss]: necessary ?

File: client/src/components/HomeSlider/HomeSlider.jsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import { Swiper, SwiperSlide } from "swiper/react";
+
+import "swiper/css";
+import "swiper/css/pagination";
+import "swiper/css/navigation";
+import "./Homeslider.scss";
+
+import { Autoplay, Pagination, Navigation, A11y } from "swiper/modules";
+
+const baseUrl = "http://localhost:3310/arcade-picture/";
```
Comment (FGAT3) [inline client/src/components/HomeSlider/HomeSlider.jsx]: Use these instead :

const BASE_URL = import.meta.env.VITE_API_URL;
const arcade-picture = `${BASE_URL}/arcade-picture/`;

File: client/src/assets/images/pacman-svgrepo-com.svg
Comment (FGAT3) [inline client/src/assets/images/pacman-svgrepo-com.svg:1]: this image should be in "server/public/assets/images/icon"

File: client/src/components/HomeSlider/HomeSlider.jsx
Comment (FGAT3) [inline client/src/components/HomeSlider/HomeSlider.jsx:1]: I think it's better to put the code directly inside HomePage than creating a component that isn't a reusable component

**Review by FGAT3 — CHANGES_REQUESTED** [review]

File: client/src/components/HomeSlider/Homeslider.scss (hunk 1 above)
Comment (pjroggy) [reply client/src/components/HomeSlider/Homeslider.scss]: done !

File: client/src/components/HomeSlider/Homeslider.scss (hunk 2 above)
Comment (pjroggy) [reply client/src/components/HomeSlider/Homeslider.scss]: you're right the background-position: center is useless !!
the following lines defined the size and position of the active bullet

File: client/src/components/HomeSlider/HomeSlider.jsx (hunk 3 above)
Comment (pjroggy) [reply client/src/components/HomeSlider/HomeSlider.jsx]: done !

File: client/src/components/HomeSlider/HomeSlider.jsx
Comment (pjroggy) [reply client/src/components/HomeSlider/HomeSlider.jsx:1]: not sure about that

File: client/src/components/HomeSlider/HomeSlider.jsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          clickable: true,
+          draggable: true,
+        }}
+        navigation
+        modules={[Autoplay, Pagination, Navigation, A11y]}
+        className="mySwiper"
+      >
+        {imageFiles.map((file) => (
+          <SwiperSlide key={file}>
+            <img src={`${arcadeUrl}${file}`} alt={`salle d'arcade ${file}`} />
```
Comment (FGAT3) [inline client/src/components/HomeSlider/HomeSlider.jsx]: Non explicit alt : "salle d'arcade arcade-room1.jpg". Maybe add alt in your imageFiles by making it a table of objects or change the name of the file on the file himself and in your table.

**Review by FGAT3 — CHANGES_REQUESTED** [review]

File: client/src/components/HomeSlider/HomeSlider.jsx (hunk 4 above)
Comment (pjroggy) [reply client/src/components/HomeSlider/HomeSlider.jsx]: done

**Review by FGAT3 — APPROVED** [review]

**Review by JadeJolivet — APPROVED** [review]

**Review by Elea641 — APPROVED** [review]

**Pull request merged**